db/migrate/20150616183237_create_receipts.rb

```diff
@@ -0,0 +1,15 @@
+class CreateReceipts < ActiveRecord::Migration
+  def change
+    create_table :receipts do |t|
+      t.references :user
+      t.text :item_name
+      t.integer :item_quantity
+      t.text :item_tags
+      t.text :item_type
+      t.text :seller_name
+      t.integer :seller_id
```

Comment:
No need to store both the seller id and the seller name, instead just store a reference like the one to user. The reference to user should also be renamed to buyer, since both buyer and seller are user models.